addons/binding/org.openhab.binding.allplay/OSGI-INF/.gitignore

```diff
@@ -0,0 +1 @@
+/*.xml
```

Comment:
What about a temporary:

```
addons/binding/org.openhab.binding.allplay/OSGI-INF/*.xml
```

imho much easier to clean up and shouts to be cleaned up 😉